### PR TITLE
feat(deprecate IE8 support): issue #13

### LINKED ISSUE #13: Drop support for IE8
Is this a feature or code bloat at this point? Are there customers of this repo that would still need support there?

### PR DESCRIPTION
**Pull request by chrisrng**
Closes #13

### CODE REVIEW

File: demo/omni-slider.js [hunk 1]
```diff
@@ -269,36 +263,19 @@ var Slider = function () {
...
-      if (this.isIE8) {
-        x = window.event.clientX + document.documentElement.scrollLeft + document.body.scrollLeft;
-        y = window.event.clientY + document.documentElement.scrollTop + document.body.scrollTop;
-      } else {
-        // If touch screen (event.touches) and if IE11 (pageXOffset)
-        x = (typeof event.clientX !== 'undefined' ? event.clientX : event.touches[0].pageX) + (window.scrollX || window.pageXOffset);
-        y = (typeof event.clientY !== 'undefined' ? event.clientY : event.touches[0].pageY) + (window.scrollY || window.pageYOffset);
-      }
+      // If touch screen (event.touches) and if IE11 (pageXOffset)
+      x = (typeof event.clientX !== 'undefined' ? event.clientX : event.touches[0].pageX) + (window.scrollX || window.pageXOffset);
```
Comment (bcVamp) [inline demo/omni-slider.js]: Non-blocking, but let's try to establish max characters per line as 80 characters, if possible going forward?

Comment (chrisrng) [inline demo/omni-slider.js]: this is compiled version from babel but im sure this happens on the file under `src` too

**Comment by MikeGertrudes** [comment]
Wow, awesome guys. Any noticeable savings in file size?

**Comment by chrisrng** [comment]
@MikeGertrudes reduced about 3KB for minified file, now it's under 10KB haha

**Comment by MikeGertrudes** [comment]
Ha, that's actually a great improvement. $9.99!

**Pull request merged**